python/pyspark/ml/wrapper.py

```diff
@@ -45,10 +45,12 @@ class JavaWrapper(Params):
 
     __metaclass__ = ABCMeta
 
-    #: The wrapped Java companion object. Subclasses should initialize
-    #: it properly. The param values in the Java object should be
-    #: synced with the Python wrapper in fit/transform/evaluate/copy.
-    _java_obj = None
```

Comment:
I don't think different instances share the same `_java_obj`. Only the default values are shared.